packages/common/src/utils/filters.ts

```diff
@@ -304,17 +304,23 @@ export const applyDefaultTileTargets = (
     return filterRule;
 };
 
-export const createDashboardFilterRuleFromField = (
-    field: FilterableField,
-    availableTileFilters: Record<string, FilterableField[] | undefined>,
-    includeDefaultValue: boolean,
-    isTemporary: boolean,
-): DashboardFilterRule =>
+export const createDashboardFilterRuleFromField = ({
+    field,
+    availableTileFilters,
+    isTemporary,
+    value,
+}: {
+    field: FilterableField;
+    availableTileFilters: Record<string, FilterableField[] | undefined>;
+    isTemporary: boolean;
+    value?: any;
+}): DashboardFilterRule =>
     getFilterRuleWithDefaultValue(
         field,
         {
             id: uuidv4(),
-            operator: FilterOperator.EQUALS,
+            operator:
+                value === null ? FilterOperator.NULL : FilterOperator.EQUALS,
```

Comment:
is this function being used somewhere where user might not create a filter like this one:
field: `name` (string)
operator: `INCLUDE`
values [null, 'irakli'] << not sure if this is possible today. just asking

Reply:
thanks for spotting this! It makes sense to rework this a bit more to avoid the bug you've just mentioned.

Reply:
thanks for the explanation. that makes sense